islandora_checksum_checker.module

```diff
@@ -278,8 +280,12 @@ function islandora_checksum_checker_validate_checksum($pid) {
               '!message' => $e->getMessage(),
             ),
             WATCHDOG_WARNING);
+          $exceptions[] = $e;
           return FALSE;
```

Comment:
This return FALSE will return before the actual new hook is invoked.

Reply:
Doh! I'm on it.

Reply:
I've invoked the hook before returning false, and also adjusted the API docs to indicate that some values may be empty. Thanks for catching that @DiegoPino.